.github/workflows/ci.yml

```diff
@@ -99,17 +99,8 @@ jobs:
           - {ARCH: 'x64', SODIUM_ARCH: 'x64', IMAGE: "windows-latest"}
         PYTHON:
           - {VERSION: "3.7", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
-          - {VERSION: "3.8", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
-          - {VERSION: "3.9", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
-          - {VERSION: "3.10", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
-          - {VERSION: "3.11", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
-          - {VERSION: "3.12", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
           - {VERSION: "3.13", NOXSESSION: "tests", SODIUM_MSVC_VERSION: "v143"}
         include:
-          - WINDOWS: {ARCH: 'arm64', SODIUM_ARCH: 'ARM64', IMAGE: "windows-11-arm"}
```

Comment:
is there a reason arm64 isn't in the matrix?

Reply:
I've fixed this (in matrix with an exclude now)